src/core/external/reveal/reveal-container.ts

```diff
@@ -19,23 +23,40 @@ import {
 import Container from '../common/container';
 import RevealElement from './reveal-element';
 
+/** This configuration handles Skyflow Reveal Elements. */
 export interface IRevealElementInput {
+  /** A Skyflow token that would be detokenised. */
   token?: string;
+  /** Redaction type of the revealed data. */
   redaction?: RedactionType;
+  /** Styles for the input in reveal element. */
   inputStyles?: object;
+  /** Label value for the reveal elements. */
   label?: string;
+  /** Styles for the label in reveal element. */
   labelStyles?: object;
+  /** An alternate text for the reveal element. */
   altText?: string;
+  /** Styles for the error text in the reveal element. */
   errorTextStyles?: object;
 }
 
+/** Provides an additional configuration options for the reveal element. */
 export interface IRevealElementOptions {
+  /** Copy icon enables revealing elements to copy text to the clipboard. It defaults to 'false'. */
```

Comment:
If \`true\` displays a copy button that copies the Reveal Element value to the clipboard. Defaults to \`false\`.